hmp/distributions.py

```diff
@@ -0,0 +1,43 @@
+class GammaDistribution():
+    def __init__(self, shape=2):
+        from scipy.stats import gamma  
+        self.shape = shape
+        self.pdf = gamma.pdf
+    def scale_to_mean(self, scale):
```

Comment:
Should have a new line between methods. Also the class and methods should have a docstring.